lib/virtmanager.pm

```diff
@@ -0,0 +1,632 @@
+package virtmanager;
+use testapi;
+
+our @ISA = qw(Exporter);
+our @EXPORT = qw(launch_virtmanager launch_xterm connection_details create_vnet create_new_pool create_new_volume clean_up_desktop create_netinterface sudo_root delete_netinterface create_guest);
+
+
+sub sudo_root {
+    type_string "sudo su -"; send_key "ret";
+    type_string get_var('ROOTPASSWORD'); send_key "ret";
+}
+
+sub launch_xterm() {
+    if (check_var("DESKTOP", "icewm") ) {
+	send_key "ctrl-alt-t", 10;
+    } elsif (check_var("DESKTOP", "gnome")) {
+	send_key "alt-f2", 10;
+	type_string "xterm\n", 50;
+    } else { 
+	# assuming default is gnome
+	send_key "alt-f2", 10;
+	type_string "xterm\n", 50;
+    }
+}
```

Comment:
x11_start_program doesn't support icewm. Icewm is the default light WM for SUSE. need to be patched.